### PR TITLE
Np 155 Use businessday

### PR DESCRIPTION
**Pull request by maxkanin**
## **Ticket**

**JIRA ID:** [NP-155](https://parking-system.atlassian.net/browse/NP-155)

## Changes proposed in this pull request
- [IMPORTANT] Reviewer **will not review** any PR without summary of changes
-implement logic handle bussiness day
please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [x] Screenshot evidence for both IOS and Android
- [x] Total change is less than 500 lines of code

## **Screenshot Evidence**

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/ef98a1f9-7a9f-4efe-bd3d-26395e06e652

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/9f9bad43-c9f0-4d81-aaf3-e3f4fca9416c

[NP-155]: https://parking-system.atlassian.net/browse/NP-155?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TaeCV** [comment]
@maxkanin Are this one still using?

**Comment by maxkanin** [comment]
ใช้ แต่ยังรอbackend bookingอยู่เลยเปิดค้างไว้

File: components/input/TimeInput.tsx [hunk 1]
```diff
@@ -77,12 +84,11 @@ const TimeInput: React.FC<TimeInputProps> = ({
 export default TimeInput;
 
 const styles = StyleSheet.create({
-  outerContainer: {
-    marginBottom: 10,
-  },
+  outerContainer: {},
```
Comment (Petoxic) [inline components/input/TimeInput.tsx]: Please remove if not use

File: components/booking/BookingDetailComponent.tsx [hunk 2]
```diff
@@ -76,37 +122,72 @@ const BookingDetailComponent: React.FC<BookingDetailComponentProps> = ({
...
-      checkOutDate == checkInDate
+      (!isFirstUpdate && checkOutDate) ||
+      (checkInTime && checkOutTime && checkInTime > checkOutTime)
     )
       setCheckOutTime(null);
-  }, [checkOutDate]);
+  }, [checkOutDate, checkInTime]);
 
+  const duration = (minTime: Date, maxTime: Date): string => {
+    return "open " + formatTime(minTime) + " - " + formatTime(maxTime);
```
Comment (Petoxic) [inline components/booking/BookingDetailComponent.tsx]: Please use \`open ${formatTime(minTime) - ${formatTime(maxTime)}\`

File: components/input/TimeInput.tsx (hunk 1 above)
Comment (maxkanin) [reply components/input/TimeInput.tsx]: done

File: components/booking/BookingDetailComponent.tsx (hunk 2 above)
Comment (maxkanin) [reply components/booking/BookingDetailComponent.tsx]: done

**Review by Petoxic — APPROVED** [review]

**Review by TaeCV — APPROVED** [review]

**Pull request merged**